tests/RestBridge.cpp

```diff
@@ -8,17 +8,19 @@
 #include <boost/test/unit_test.hpp>
 #include <restbridge/RestBridge.h>
 
-#include <lunchbox/rng.h>
+#include <random>
 
+static std::mt19937_64 rng;
 
 BOOST_AUTO_TEST_CASE( test_construction )
 {
+    rng.seed(60000);
     BOOST_CHECK_NO_THROW( restbridge::RestBridge( "localhost", 12345 ));
 }
 
 BOOST_AUTO_TEST_CASE( test_invalid_run )
 {
-    const unsigned short port = (lunchbox::RNG().get<uint16_t>() % 60000) + 1024;
+    const unsigned short port = rng() + 1024;
```

Comment:
You also want to seed your RNG with random data, otherwise you'll get port clashes again since same-seeded PRNGs returns same random values.